DOCS-8161 CloudHub Required Permission (#783)
mulesoft/docs-connectors: modules/ROOT/pages/cloudhub/cloudhub-connector-ref.adoc

CloudHub Connector v1.0

> Anypoint Connector for CloudHub enables you to create and list notifications, and to retrieve the status of created applications that are deployed to CloudHub.

Reviewer: ```suggestion
Anypoint Connector for CloudHub enables you to create and list notifications and to retrieve the status of created applications that are deployed to CloudHub.
```

Reviewer: Following edits for parallelism:
<blah> | <verb phrase (configures, specifies, enables, rejects....

| Name | Type | Description | Default Value | Required
| Username a| String |  Username of the Anypoint Platform Account |  |x
| Password a| String |  Password of the Anypoint Platform Account |  |x

> | Environment a| String |  (Optional) Anypoint Platform environment to use. By default the enviroment defined as the default in Anypoint Platform is used. |  |

Reviewer: ```suggestion
| Environment a| String |  Optionally specifies the Anypoint Platform environment to use: by default, the environment defined as the default in Anypoint Platform. |  |
```

> | Anypoint Platform URL a| String |  This is used to change between the US and EU Anypoint Platform if it is required. |  `+https://anypoint.mulesoft.com+` |

Reviewer: ```suggestion
| Anypoint Platform URL a| String |  Enables you to change between the US Anypoint Platform and the EU Anypoint Platform, if required. |  `+https://anypoint.mulesoft.com+` |
```

> | Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy |  |

Reviewer: ```suggestion
| Reconnection a| <<Reconnection>> |  If `true`, rejects deployment if the connector connectivity test is unsuccessful after exhausting the associated reconnection strategy. |  |
```
